README.md

```diff
@@ -25,14 +28,17 @@ sce link transit
       name: Milpitas
       id: MLPT
   grouped_stops:
-    - operator: CT
-      group_name: San Jose Diridon Caltrain Station
-      ids:
-        - 70261
-        - 70262
+    - name: San Jose Diridon Caltrain Station
+      stops:
+        - operator: CT
+          id: 70261
+        - operator: CT
+          id: 70262
```

Comment:
can we undo the newlines that were added to this file, the only changes should be this section